configs/envoy_double_proxy_v2.template.yaml

```diff
@@ -32,6 +32,8 @@
         "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
         codec_type: AUTO
         stat_prefix: router
+        common_http_protocol_options:
+          idle_timeout: 840s
```

Comment:
How did this pass tests in the first place? Shouldn't this have failed somehow on the compile time options build?

Reply:
We had the example configs test tagged as excluded.  I'll go and fix that as I think it's more trouble than it's worth.